benchopt/plotting/plot_objective_curve.py

```diff
@@ -84,42 +76,28 @@ def plot_objective_curve(df, obj_col='objective_value', plotly=False,
 
         q1 = df_.groupby('stop_val')['time'].quantile(.1)
         q9 = df_.groupby('stop_val')['time'].quantile(.9)
-
+        
         fill_between_x(
-            fig, curve['time'], q1, q9, curve[obj_col], color=CMAP(i % CMAP.N),
-            marker=markers[i % len(markers)], label=solver_name, plotly=plotly
+            fig, curve['time'], q1, q9, curve[obj_col], color=colors[i % CMAP.N],
+            marker=markers[i % len(markers)], label=solver_name, plotly=False
```

Comment:
remove argument `plotly` from `fill_between_x` and possibly simplify. if this is two lines, consider removing the finc as I believe it is not reused anywhere?

Reply:
No it is not, neither is `add_h_line()`so I will remove these two functions here and remove the `if not plotly` part in their definition.